<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugViewer</class>
 <widget class="QDialog" name="DebugViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>428</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Debugger</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QPushButton" name="debugGetHexValues">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>210</y>
        <width>201</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Hex Values</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="debugAddress">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="debug8Bit">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>60</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>8bit</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="debug16Bit">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>60</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>16bit</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="debug32Bit">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>60</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>32bit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="debugLength">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>110</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>90</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Length</string>
      </property>
     </widget>
     <widget class="QListWidget" name="debugValueList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>401</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="debugDump">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>310</y>
        <width>201</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Dump To File</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="debugFilePath">
      <property name="geometry">
       <rect>
        <x>432</x>
        <y>160</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>140</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>File Path</string>
      </property>
     </widget>
     <widget class="QPushButton" name="debugShowRegisters">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>230</y>
        <width>201</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Registers</string>
      </property>
     </widget>
     <widget class="QPushButton" name="debugPrintDebugLogs">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>250</y>
        <width>201</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Print Debug Logs</string>
      </property>
     </widget>
     <widget class="QPushButton" name="debugEraseDebugLogs">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>270</y>
        <width>201</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Erase Debug Logs</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DebugViewer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DebugViewer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
